locales/fr/fr.json

```diff
@@ -62,11 +62,11 @@
     "Are you sure you want to detach this resource?": "Êtes-vous sûr(e) de vouloir détacher cette donnée ?",
     "Are you sure you want to force delete the selected resources?": "Êtes-vous sûr(e) de vouloir forcer la suppression des données sélectionnées ?",
     "Are you sure you want to force delete this resource?": "Êtes-vous sûr(e) de vouloir forcer la suppression de cette donnée ?",
-    "Are you sure you want to log out?": "Are you sure you want to log out?",
+    "Are you sure you want to log out?": "Êtes-vous sûr de vouloir vous déconnecter ?",
```

Comment:
sûr(e)

Reply:
Le masculin l'emporte en grammaire donc pour ici "sûr" est correct.

Reply:
Partout, nous utilisons sûr(e). Autant rester homogène

Reply:
Hi @caouecs , I have fix it